src/Footer.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+
+class Footer extends React.Component {
+  constructor(props) {
```

Comment:
Convert this component to functional component

Reply:
Done